### ISSUE TITLE
Cult of Remute Portable (DS Homebrew) freezes when started through TwilightMenu++ on DSi

### ISSUE THREAD

**Issue by NeitherMeNorYou** [DS-Homebrew/nds-bootstrap#1478]
**TL;DR:** The DS homebrew "Cult of Remute Portable", which is a music album created by the artist Remute over [on Bandcamp](https://remute.bandcamp.com/music), freezes my DSi after a few moments of playing any track when started via TwilightMenu++. Running the ROM from an R4 card will allow it to play perfectly on the same device.

### ROM:

Directly from the artist's bandcamp page ([here](https://remute.bandcamp.com/album/the-cult-of-remute-portable-rom-file-for-nintendo-ds)), and I've double checked the checksum to ensure it was a proper download.

### DSi:
- Unlaunch 2.0
- TwilightMenu++ v25.4.0 (full release from the github page)
- nds-bootstrap v0.65.1 (from above's TwilightMenu++ release)
- booting the rom from the SD Card

As far as I can tell all settings regarding TwilightMenu++ and nds-bootstrap are at default for the rom.

### Expected Behavior:

The homebrew should launch like any other .nds rom and then just start playing the music until turned off.

### Actual Behavior:

The homebrew starts, music starts playing, and then after a short while the DSi freezes.

### Steps to reproduce:

- Boot rom from TwilightMenu++
- let it start playing
- after about 5-15 seconds it starts seizing up

Very rarely it will take longer, like 30 seconds, but it will freeze eventually. No interaction is required with the rom, and pressing buttons doesn't change anything either.

### Misc information:

Adding the rom to my R4 card and starting it from there will allow my DSi to play the rom perfectly, with no crashes.

**Comment by Epicpkmn11** [comment]
Can you try loading it from [GodMode9i](https://db.universal-team.net/ds/godmode9i) instead of TWiLight Menu++ to see if that changes anything?

You can load GM9i from TWiLight, then just load this from there.

**Comment by NeitherMeNorYou** [comment]
I'm not sure if I did it the "correct" way, since I haven't used GodMode9i on my DSi yet, but it still freezes.

Just for you to double check what I did, I downloaded Godmode9i.dsi from the link you provided, added it to the root of my SD card, then started it from TwilightMenu++. Once gm9i was loaded into the menu I started the rom using "Boot file (Direct)".

Using Unlaunch it just displays a white screen forever and does not boot into the rom itself, though again not sure if I have to do anything extra, I just selected the rom and pressed A.

**Comment by Epicpkmn11** [comment]
Hmm, seems like the ROM probably doesn't natively support DSi SD then, so this is indeed an nds-bootstrap issue not TWiLight Menu++.

I'm afraid I'm not sure how to fix it then, @RocketRobz might have more ideas.

**Comment by Epicpkmn11** [comment]
Actually one idea, in TWiLight Menu++ press <kbd>Y</kbd> on the game and change `Run in` to `DS Mode` and see if that helps.

**Comment by NeitherMeNorYou** [comment]
Starting it in DS mode from TwilightMenu++ doesn't work at all, it just stays on a pure white screen indefinitely.

If it's relevant, the TwilightMenu++ settings for Load Bootloader are on "Thru nds-bs" and Bootstrap is set to "Release".

**Comment by NeitherMeNorYou** [comment]
I'm sad to say that it does not appear to have been fixed yet.

Just tried it on the latest release of TWLM++ (27.5.2,) and it still crashes in DSi mode after about a minute and stays on a white screen permanently when trying to load it in DS mode.

If you'd like I could send you a kofi to pay for the rom, if you think that might help.
